Composer/packages/client/src/recoilModel/dispatchers/utils/project.ts

```diff
@@ -1,12 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import path from 'path';
```

Comment:
You sure you can use this module on the browser?

Reply:
Great catch. @hatpick and me had an offline conversation about this.

#4437 This would be the ticket that should do a cleanup of the path module throughout our codebase in non node context